scripts/hostcfgd

```diff
@@ -781,11 +792,29 @@ class AaaCfg(object):
                 interface_ip = ipv4_addr
         return interface_ip
 
+
+    def check_file_integrity(self, filename):
+        rc, out = exec_cmd(cmd="du -b {}".format(filename), verbose=False)
```

Comment:
>exec_cmd [](http://example.com/codeflow?start=18&length=8)

This is python. So prefer python API than equivalent command line invoke

Reply:
Fixed.